public/docs/ts/latest/cookbook/a11y.jade

```diff
@@ -0,0 +1,444 @@
+include ../_util-fns
+
+:marked
+  ## Welcome to A11y!
+
+  You are about to learn how to make your Angular&nbsp;2 application accessible for
+  as many people as possible. This is no small goal, there is a large group of people out there who find it
+  very hard or even impossible to use applications that has not been built with these concepts in mind.
+
+  We will show you that these concepts can be integrated with little to no extra effort on your part,
+  provided you take them into account right from the design phase of your application.
+
+  So page through this cookbook, apply the recipes contained herein and watch your userbase grow.
+
+  If you are seeing the terms `Web Accessibility`, `ARIA` or `a11y` for the first time, you are at the start of an
+  incredible journey. And for those who already enjoy the benefits of `Accessible Web Applications`, we will
+  show you how to apply your knowledge in the new and exciting world of Angular&nbsp;2.
+
+.l-main-section
+:marked
+  ## A11y and ARIA in a nutshell
+
+  `Accessibility` is often referred to as `a11y`. This is because we want to say as lot while we write less. And
+  as it has eleven letters, starts with an `a` and ends with a `y`, we shorten this word to `a11y`. We will
+  refer to `a11y` when we actually want to say `accessibility`.
+
+.l-sub-section
+  :marked
+    In short, `a11y` refers to creating web applications that everyone can use. Thus making it accessible to everyone.
+
+:marked
+  If you are totally new to the term you may want to have a look at what the folks at the `W3C` have to say about
+  [a11y](https://www.w3.org/WAI/intro/accessibility.php), to put the rest of the article into perspective.
+
+  But what is that other word?
+
+.l-sub-section
+  :marked
+    `ARIA`, or `Accessible Rich Internet Applications` refers to bringing `a11y` concepts into
```

Comment:
Worth mentioning that it is a standard set of attributes for adding accessibility information to HTML and SVG.